doc/sphinx/source/esmvaldiag/git_repository.rst

```diff
@@ -163,15 +162,15 @@ Do-s
 * Use short but self-explanatory variable names (e.g., model_input and reference_input instead of xm and xr).
 * Consider a modular/functional programming style. This often makes code easier to read and deletes intermediate variables immediately. If possible, separate diagnostic calculations from plotting routines.
 * Consider reusing or extending existing code. General-purpose code can be found in esmvaltool/diag_scripts/shared/.
-* Comment all switches and parameters including a list of all possible settings/options in the header section of your code (see also Section :ref:`std_diag`).
-* Use templates for recipes (Section :ref:`std_recipe`) and diagnostics (Section :ref:`std_diag`) to help with proper documentation.
+* Comment all switches and parameters including a list of all possible settings/options in the header section of your code (see also ...).
+* Use templates for recipes (see ...) and diagnostics (see ...) to help with proper documentation.
 * Keep your *FEATURE BRANCH* regularly synchronized with the *DEVELOPMENT BRANCH* (git merge).
 * Keep developments / modifications of the ESMValTool framework / backend / basic structure separate from developments of diagnostics by creating different *FEATURE BRANCHES* for these two kinds of developments. Create *FEATURE BRANCHES* for changes / modifications of the ESMValTool framework only in the *PUBLIC* repository.
 
 Don't-s
 -------
 
-* Do not use other programming languages than the ones currently supported (NCL, Python, R). Contact the Core Development Team (Section :ref:`core_dev_team`) if you wish to use another language, but remember that only open-source languages are supported by the ESMValTool.
+* Do not use other programming languages than the ones currently supported (NCL, Python, R). Contact the :doc:`ESMValTool Core Development Team <../preface/core_team>` if you wish to use another language, but remember that only open-source languages are supported by the ESMValTool.
```

Comment:
```suggestion
* Do not use other programming languages than the ones currently supported (Python, R, NCL, Julia). If you are unsure what language to use, Python is probably the best choice, because it has very good libraries available and is supported by a large community. Contact the :doc:`ESMValTool Core Development Team <../preface/core_team>` if you wish to use another language, but remember that only open-source languages are supported by the ESMValTool.
```